blogposts/announcing-sourcegraph-3.13.md

```diff
@@ -0,0 +1,238 @@
+
+## Create campaigns to run arbitrary code over all your repositories using the `src-cli`
+
+<p class="container">
+  <div style="padding:56.25% 0 0 0;position:relative;">
+    <iframe src="https://player.vimeo.com/video/390882141?color=0CB6F4&amp;title=0&amp;byline=" style="position:absolute;top:0;left:0;width:100%;height:100%;" frameborder="0" webkitallowfullscreen="" mozallowfullscreen="" allowfullscreen=""></iframe>
+  </div>
+  <p style="text-align: center"><a href="https://vimeo.com/390882141" target="_blank">View on Vimeo</a></p>
+</p>
+
+Sourcegraph code change campaigns now support running arbitrary code over all your repositories. Using the `src-cli`, you will generate a diff from customized steps. This means that you can now run series of steps in your campaigns, such as a find and replace action, then running a linter over the changed code. 
+
+Simply provide an `action.json` with a scope query and command steps. The CLI will programmatically download zips of each of your repositories that match the scope query, and apply each command to each of those repositories. The result will be a diff to be uploaded to Sourcegraph for previewing and creating a campaign.
+
+For example, if I wanted to bump the version of a library my `action.json` would be:
+
+```json
+
+TO DO
+
+```
+
+GitHub labels associated with campaigns are now displayed. Support for Bitbucket Server labels is coming soon. 
+
+When creating a campaign, users can now specify the branch name that will be used on code host. This is also a breaking change for users of the GraphQL API since the `branch` attribute is now required in `CreateCampaignInput` when a `plan` is also specified.
+
+## Basic code intelligence support reaches 32 languages
+
+![Sourcegraph basic code intelligence languages](/images/3-13-language-extensions.png "Sourcegraph basic code intelligence languages")
+
+With the addition of GraphQL, Groovy, Objective-C, Pascal, Protobufs, Verilog, and VHDL, Sourcegraph basic code intelligence now supports 32 languages. Every Sourcegraph instance comes enabled with basic code intelligence provided by search based heuristics. For fast and precise code intelligence see our [LSIF documentation](https://docs.sourcegraph.com/user/code_intelligence/lsif).
+
+## LSIF-based precise code intelligence for Dart
+
+TODO: screenshots
```

Comment:
@efritz Can you recommend a repo with LSIF data for showing Dart code intel?

Reply:
Working on this now: https://sourcegraph.slack.com/archives/CHXHX7XAS/p1582212428372300

Reply:
In the google doc he recommended https://sourcegraph.com/github.com/sourcegraph/lsif-dart, but it's not currently showing hovers. Thread about this in slack: https://sourcegraph.slack.com/archives/CHXHX7XAS/p1582212428372300

Reply:
Added a quick video. @ryan-blunden feel free to update later if you have a better one in mind!